extensions/amp-script/amp-script.md

```diff
@@ -142,6 +142,24 @@ AMP.getState(expr) {}
 </script>
 ```
 
+### Exporting functions for use in <amp-list>
+
+You may export functions to act as the data source for an `<amp-list>`.
+The exported function must return either JSON, or a Promise which will eventually contain JSON.
+
+The export API is available on the global scope, and has the following signature:
+
+```js
+/**
+ * @param {string} name the name to identify the function by.
+ * @param {Function} function the function to export.
+ */
+function exportFunction(name, function) {}
+```
+
+**nodom (optional)**
+There are situations in which you may wish to use `<amp-script>` as solely a data-layer, rather than using it to manipulate dom. The `nodom` attribute removes the ability for the `<amp-script>` to make dom modifications, in favor of a signficantly smaller bundle size and therefore better performance. It also renders the `<amp-script>` invisible, removing the need for specifying height and width.
```

Comment:
This should be in the attributes section.

Reply:
let's capitalize "DOM" too.

Reply:
whoops, I moved it to the wrong place while fixing a merge conflict 🤕